apps/condo/lang/ru/ru.json

```
@@ -3478,5 +3464,19 @@
   "ticket.problem.classifier.PEST_CONTROL_DISINFECTION.name": "Дератизация, дезинсекция, дезинфекция",
   "ticket.problem.classifier.CRACKS_MASONRY_PLASTER_DAMAGE.name": "Трещины, выпадение кладки/штукатурки, повреждение отмостки/цоколя",
   "ticket.problem.classifier.SIGNAGE_ADVERTISING_EQUIPMENT_ON_FACADE.name": "Надпись/реклама/оборудование на фасаде",
-  "ticket.problem.classifier.REPAIR_IN_ROOM.name": "Ремонт в помещении"
+  "ticket.problem.classifier.REPAIR_IN_ROOM.name": "Ремонт в помещении",
+  "ticket.place.classifier.IT_SERVICES.name": "IT сервисы",
+  "ticket.place.classifier.COURTYARD.name": "Двор",
+  "ticket.place.classifier.APARTMENT.name": "Квартира",
+  "ticket.place.classifier.RECEIPT.name": "Квитанция",
+  "ticket.place.classifier.ENTRANCEWAY.name": "Подъезд",
+  "ticket.place.classifier.PAVEMENTS_ROADS_LAWNS.name": "Тротуары, дороги, газоны",
+  "ticket.place.classifier.DOCUMENTS.name": "Документы",
+  "ticket.place.classifier.FACADES.name": "Фасады",
+  "ticket.place.classifier.ATTICS_BASEMENTS.name": "Чердаки, подвалы",
+  "ticket.place.classifier.SECURITY.name": "Безопасность",
+  "ticket.place.classifier.ROOFING.name": "Кровля",
+  "ticket.place.classifier.NON_RESIDENTIAL_PREMISES.name": "Нежилое помещение",
+  "ticket.place.classifier.PARKING.name": "Паркинг",
+  "ticket.place.classifier.FEEDBACK_ON_WORK.name": "Отзыв о работе"
```

```suggestion
  "ticket.place.classifier.PERFORMANCE_REVIEW.name": "Отзыв о работе"
```

(optional)
more aligned to me

---

sounds too it-ish

---

Maybe we'll do it "WORK_FEEDBACK". To be sync with the field in the ticket?